homework1/task2/task2.py

```diff
@@ -0,0 +1,28 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+    please write function "check_fib", which accepts a Sequence of integers, and
+    returns if the given sequence is a Fibonacci sequence
+We guarantee, that the given sequence contain >= 0 integers inside.
+"""
+import math
+from typing import Sequence
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+    def is_fib_num(n: int) -> bool:
```

Comment:
so u found this solution in google, copied it, but decided not to use it anywhere?